lib/scope.js

```diff
@@ -162,16 +168,21 @@ function defineScope(cls, targetClass, name, params, methods, options) {
           return definition.related(self, f._scope, condOrRefresh, cb);
         }
       };
-      
+
       f._receiver = this;
       f._scope = typeof definition.params === 'function' ?
         definition.params.call(self) : definition.params;
-      
+
       f._targetClass = targetModel.modelName;
       if (f._scope.collect) {
         f._targetClass = i8n.camelize(f._scope.collect);
       }
-      
+
+      f.get = function (refresh) {
+        var cb = utils.createPromiseCallback();
+        return definition.related(self, f._scope, refresh, cb);
```

Comment:
OK, but note that all the `related` functions default to `refresh=false` if refresh is omitted.  
No idea why...